specification/resources/resource-manager/Microsoft.Resources/stable/2021-04-01/resources.json

```diff
@@ -4568,7 +4568,10 @@
           "description": "The URI of the template. Use either the templateLink property or the template property, but not both."
         },
         "parameters": {
-          "type": "object",
+          "type": "array",
```

Comment:
shouldn't this be object?

Reply:
Isn't this not just an individual parameter (the value of value is mistakenly an object) but the whole _collection_ of these parameters? Hence parameterS (plural). 

In other words, the top-level property in the following hierarchy:

```
parameters
  <parameter name>
    value
```

Reply:
... which is a dictionary (object), not an array

Reply:
Oh, indeed! Me stupid, me fix it.